.github/workflows/api-integration-tests.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     strategy:
       matrix:
         php-versions: ['7.4', '8.0']
-        nextcloud: ['stable23', 'stable24', 'stable25']
+        nextcloud: ['stable24', 'stable25']
```

Comment:
Can we do 26 already? Beta 1 is out

Reply:
that branch does not exist yet, I guess they only create it on final release or some RC